UnitTests/ComponentViewTests.swift

```diff
@@ -262,4 +262,8 @@ class ComponentViewTests: XCTestCase {
     func testObjectPagePriceView() {
         snapshot(.objectPagePriceView)
     }
+
+    func testLinkButtonListView() {
+        snapshot(.linkButtonListView)
```

Comment:
check the snapshot image, it seem there is an alignment issue between the button and subtitle (check #778 i had that issue too) 

<img width="505" alt="Screenshot 2020-02-10 at 14 59 59" src="https://user-images.githubusercontent.com/167989/74156180-201d3d00-4c16-11ea-8c94-f10bf09c5503.png">

Reply:
What was the solution? I can't figure out what's causing this in the firsst place 🙈

Reply:
_For future reference_

The solution was to set `button.contentHorizontalAlignment = .leading`.